Makefile

```diff
@@ -61,9 +62,8 @@ PKGNAME = github.com/hyperledger/$(PROJECT_NAME)
 
 METADATA_VAR = Version=$(PROJECT_VERSION)
 
-# hard-code GO_VER until CI supports Go 1.12.9
-#GO_VER = $(shell grep "GO_VER" ci.properties |cut -d'=' -f2-)
-GO_VER ?= 1.12.9
+# Unit tests fail when upgrading to Go 1.12.9
```

Comment:
There's a PR to address that so we should get that PR in and remove this.

Seems like ci.properties may also be able to go away in a different commit/PR

Reply:
Since ci.properties isn't used by AZP I've updated the value in there so I can remove the hardcoded value and this comment. I'll do another PR with the removal of ci.properties and once your PR gets in, I will update CI to 1.12.9

Reply:
The other PR was merged I will integrate those changes